proposal_stages.md

```diff
@@ -0,0 +1,28 @@
+# ROS 2 Production Proposal Stages
+The following stages are also intended to be used as github tags for tracking.
+## Stage 1: Draft
+
+The draft stage represents the initial proposal of a project or task which has been formalised into a github issue for presentation to the community. To move to the next stage, this draft should be presented to the ROS2 Production WG and discussed in relevant channels (e.g slack, github, discourse, etc.), having addressed feedback about the scope, methodology, and tasks required to implement this proposal. 
+
+## Stage 2: Ready
+
+In this stage, the draft has received feedback from the community. It is considered to be in “ready for work” state. Tasks have been clearly defined and broken down into relevant sub-components and subtasks, sufficient such that a principal contributor may begin execution with clear direction. To move to the next stage, there should be a discussion/vote on the proposal’s priority, who will be the principal contributor, and the intended timeline for completion. 
+
+
+## Stage 3: Accepted
+
+In this stage, the proposal has been accepted and is part of the ROS 2 Production WG’s project planning board. A principal contributor has been identified. Accepted proposals will be visible on the working group’s project roadmap. To move to the next stage, the principal contributor should be providing regular updates to the working group. 
+
+## Stage 4: In Progress
+
+In this stage, the proposal is in the middle of development and is receiving updates regularly from contributors. For in-progress proposals, they will be updated to the wider technical steering committee. To move to the next stage, the proposal work should be near completion and receiving final reviews.
+
+## Stage 5: In Review
+
+In this change, other community members/developers are reviewing the implementation and quality of the proposal/task work, with minor* changes, if any, that may be required for completion. To move to the next stage, all issues from reviewers have been addressed.
+
+## Stage 6: Done
+
+This is the final stage. All relevant subtasks and issues have been addressed. If there is additional maintenance work, this can be considered separate from the initial feature or proposal implementation.
+
```

Comment:
Stage 7 : deferred/stale?

Reply:
Is this state branched from a proposal that has yet to be accepted (e.g. not getting any clear vision or the contributor stops addressing feedback) or is it for work that was already accepted and in progress, however no progress is being made after a period?